tool/tsh/tsh.go

```diff
@@ -1537,7 +1550,7 @@ func onSCP(cf *CLIConf) error {
 
 // makeClient takes the command-line configuration and constructs & returns
 // a fully configured TeleportClient object
-func makeClient(cf *CLIConf, useProfileLogin bool) (*client.TeleportClient, error) {
+func makeClient(cf *CLIConf) (*client.TeleportClient, error) {
```

Comment:
I don't think we can safely remove this parameter. Just pass `false` here from where you need to call it - you only need to call Ping endpoint so HostLogin will be unused anyway.